.github/workflows/release.yml

```diff
@@ -0,0 +1,115 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      prerelease:
+        description: "Prerelease"
+        required: true
+        default: false
+        type: boolean
+      draft:
+        description: "Draft"
+        required: true
+        default: false
+        type: boolean
+
+permissions:
+  contents: write
+
+jobs:
+  release:
+    name: Version Bump and Release
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.AUTOMATION_USER_TOKEN }}
+          fetch-depth: 0
+
+      - name: Get latest non-prerelease release
+        id: latestrelease
+        uses: cardinalby/git-get-release-action@v1
+        with:
+          latest: true
+          prerelease: false
+          draft: false
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Get next version
+        id: semverbump
+        uses: WyriHaximus/github-action-next-semvers@v1.2.1
+        with:
+          version: ${{ steps.latestrelease.outputs.tag_name }}
+
+      - name: Update Version in code
+        run: |
+          sed -i "s/^version = \"[0-9].[0-9].[0-9]\"/version = \"${{steps.semverbump.outputs.patch}}\"/g" build.gradle
+          sed -i "s/private String sdkVersion = \"[0-9]\.[0-9]\.[0-9]\";/private String sdkVersion = \"${{steps.semverbump.outputs.patch}}\";/g" src/main/java/com/devcycle/sdk/server/common/model/PlatformData.java
+          sed -i "s/java-server-sdk:[0-9]\.[0-9]\.[0-9]/java-server-sdk:${{steps.semverbump.outputs.patch}}/g" ./example-cloud/build.gradle
+          sed -i "s/java-server-sdk:[0-9]\.[0-9]\.[0-9]/java-server-sdk:${{steps.semverbump.outputs.patch}}/g" ./example-local/build.gradle
+          sed -i "s/<java\.server\.sdk\.version>[0-9]\.[0-9]\.[0-9]<\/java\.server\.sdk\.version>/<java.server.sdk.version>${{steps.semverbump.outputs.patch}}<\/java.server.sdk.version>/g" ./benchmark/pom.xml
+          sed -i "s/<version>[0-9]\.[0-9]\.[0-9]<\/version>/<version>${{steps.semverbump.outputs.patch}}<\/version>/g" ./README.md
+          sed -i "s/java-server-sdk:[0-9]\.[0-9]\.[0-9]/java-server-sdk:${{steps.semverbump.outputs.patch}}/g" ./README.md
```

Comment:
Think we're defaulting to updating the minor version currently in the release action, so this won't match up.

Reply:
oh, good to know.  I'll update it